deployment/helm/README.md

```diff
@@ -0,0 +1,58 @@
+# Helm
+
+`reactive-interaction-gateway` is an example [helm](https://helm.sh/) chart for Kubernetes, configured to run RIG in distributed mode.
+
+## Configuration
+
+### Docker image
+
+You can use your own Docker image if it's based on original RIG Docker image.
+
+```
+image: accenture/reactive-interaction-gateway
+```
+
+### NODE_HOST
+
+IP used to set Node host for each node. Automatically taken from the environment variable ```NODE_HOST``` within the deployment configuration.
+
+```
+- name: NODE_HOST
+  valueFrom:
+    fieldRef:
+      fieldPath: status.podIP
+```
+
+### DNS_NAME
+
+Address upon which RIG will do DNS discovery. This has to point to Kubernetes headless service.
+
+```
+- name: DNS_NAME
+  value: "reactive-interaction-gateway-service-headless.default.svc.cluster.local"
+```
+
+## Running locally with Minikube
+
+If you want to use local image (either original RIG or your own based on original RIG):
+
+1. Switch to Minikube context `eval $(minikube docker-env)`
+1. Build your image `docker build -t rig .`
+1. Check `docker images` => new image should be listed there
+1. Supply the image name to the [values file](reactive-interaction-gateway/values.yaml) e.g. `repository: rig` (line 8)
+1. Add `imagePullPolicy: Never` under line 10 [values](reactive-interaction-gateway/values.yaml)
+
+
+## Start RIG on Kubernetes
+1.  Make sure a tiller pod is running in the kube-system namespace ```kubectl get pods -n=kube-system```
+1. `helm install reactive-interaction-gateway` (assuming you are in root directory)
```

Comment:
This is true if I'm in `deployment/helm` folder, from root I need to use `helm install deployment/helm/reactive-interaction-gateway/`. Please update it to one way or another.